modules/videoio/include/opencv2/videoio.hpp

```diff
@@ -650,6 +650,14 @@ class CV_EXPORTS_W VideoCapture
     */
     CV_WRAP virtual bool open(int index);
 
+    /** @brief  Open a camera for video capturing
+
+    @overload
+
+    Parameters of the methods are open(int cameraNum, int apiPreference),@return open(cameraNum + apiPreference).
```

Comment:
Also link should be on constructor VideoCapture instead this function itself. Use method above as an example.